controllers/core/openstackcontrolplane_controller.go

```diff
@@ -248,6 +245,16 @@ func (r *OpenStackControlPlaneReconciler) reconcileNormal(ctx context.Context, i
 		return ctrlResult, nil
 	}
 
+	// Reconcile optional services when enabled. If the service is explicitly disabled,
+	// we shouldn't try to reconcile it.
+	if instance.Spec.Cinder.Enabled {
```

Comment:
this is basically not really required, right? the current approach would do the same. and just have the condition to initialize the unknown state if enabled, or have I missed something?

Reply:
Yeah, this isn't necessary since we check for the enabled status in `pkg/openstack/cinder.go`. But I removed it from cinder.go to avoid calling reconcile at all. I just commented on that, acknowledging it might not be a desirable approach.
https://github.com/openstack-k8s-operators/openstack-operator/pull/162#discussion_r1088641451

I'll update the PR and remove this to leave the established pattern in place. ...incoming update.